package.json

```diff
@@ -25,6 +25,7 @@
     "build-components": "lerna run --scope web-components build --stream",
     "build-storybook": "yarn build-components && lerna run build-storybook --stream",
     "build-auth": "yarn build-components && lerna run --scope web-auth build --stream",
+    "clean": "find . -iname 'node_modules' -maxdepth 2 | xargs rm -rf",
```

Comment:
Intentional? sounds useful

Reply:
yup, i wanted to add this in because it's come up a few times for me. requires two system level dependencies though `find` and `xargs` (i think these are quite common though).

Reply:
@haveanicedavid mentioned that he might want to delete other build files as well. maybe we can do that in the future and just chain together various `clean` commands. so eventually we'd have something like:

```
"clean": "yarn clean-build-files && yarn clean-node-modules"
```